docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/ToSchemaReference.scala

```diff
@@ -23,6 +23,10 @@ private[schema] class ToSchemaReference(
       if (originalSchema.deprecated != schema.deprecated && schema.deprecated) result = result.copy(deprecated = Some(schema.deprecated))
       if (originalSchema.attributes.get(Title.Attribute) != schema.attributes.get(Title.Attribute))
         result = result.copy(title = schema.attributes.get(Title.Attribute).map(_.value))
+      if (
+        originalSchema.attributes.get(Nullable.Attribute).exists(_.nullable) || schema.attributes.get(Nullable.Attribute).exists(_.nullable)
```

Comment:
I think we should copy over the attribute verbatim, not only when it's set to `true`

Reply:
I assume this should just check whether the attribute exists now that this is a `Unit` attribute? :smile:

Reply:
yeah, there are already two states: exists and does not exist